scripts/apps/monitoring/controllers/MonitoringController.js

```diff
@@ -61,4 +80,18 @@ export function MonitoringController($rootScope, $location, desks) {
         vm.singleGroup.singleViewType = null;
         vm.singleGroup = null;
     }
+
+    function switchView(viewColumn) {
```

Comment:
We need to document _all_ functions in the codebase. This will soon be in our contributions guidelines which are in the making